app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      log_in @user
```

Comment:
I think we met `remember user` in this line.
Please fix this before moving on to get rid of `undefined method`

Reply:
Thanks. Its being implemented now.